__tests__/utils/ethSigner.test.ts

```diff
@@ -354,12 +355,25 @@ describe('Ethereum signer', () => {
 describe('Ledger Signer', () => {
   // signature of Ledger can't be tested automatically.
```

Comment:
Maybe there is some way with virtual ledger like
https://developers.ledger.com/docs/ledger-live/discover/integration/wallet-api/simulator/introduction

Reply:
Seems not to be very reliable. The dev (a French guy) that coded the APP has no NanoX, tested it with success Speculos simulator.
On my side, I found several problems with my own NanoX. So, this simulator is useful for inital tests, but is not very reliable...